src/core/server/elasticsearch/client/configure_client.ts

```diff
@@ -98,10 +98,16 @@ function getResponseMessage(event: RequestEvent): string {
 const addLogging = (client: Client, logger: Logger) => {
   client.on('response', (error, event) => {
     if (event) {
+      const opaqueId = event.meta.request.options.opaqueId;
+      const meta = opaqueId
+        ? {
+            http: { request: { id: event.meta.request.options.opaqueId } },
+          }
+        : undefined; // do not clutter logs if opaqueId is not present
```

Comment:
`"request": { }` adds noise to the logs and consumes more disk space, but doesn't bring any value.